terra-jupyter-aou/Dockerfile

```diff
@@ -126,7 +126,7 @@ RUN echo "Sys.setenv(RETICULATE_PYTHON = '$(which python3)')" >> ~/.Rprofile
 
 # Install Notebook libraries as the user.
 
-ENV USER jupyter-user
+ENV USER jupyter
```

Comment:
Revert AoU changes?

Reply:
I was thinking we can keep the aou change except version is not bumped, so no new image is generated...and I'll try working on a separate PR for aou to get tests to work...but I guess I can also just revert this, since it's pretty small change